source/reference/verification/hash.txt

```diff
@@ -0,0 +1,98 @@
+.. _c2c-verify-hash-comp:
+
+===========================
+Verify with Hash Comparison
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can verify sync by comparing MD5 hashes of collections synced from the
+source cluster to the destination cluster. While hash comparison ensures
+that the destination cluster has received all changes from the source, the
+:dbcommand:`dbHash` command locks the cluster, preventing additional writes
+until it completes.
+
+.. note::
+
+   Hash comparison is not possible with sharded clusters. It also does not
+   work for standalone servers and replica sets that use MongoDB 4.4 or earlier
+   releases, since the document field order can vary.
+
+If you need to sync a sharded cluster or an older release of MongoDB, or
```

Comment:
This paragraph also feels like it might work nicely in an About this Task section following the copy from the admonition.

Reply:
Fixed